<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer</class>
 <widget class="QDialog" name="Customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">    background-color: #5AB2FF;</string>
  </property>
  <layout class="QFormLayout" name="formLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="styleSheet">
      <string notr="true">border: 2px solid #1E0342;
background-color: #3572EF;
color: #1E0342;
font:30px;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_Name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_Name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Pass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_Password">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_CPass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_Confirm_Password">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_NN">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>National Number</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_NN">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_Age">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEdit_Age">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_Address">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_Address">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_Phone">
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lineEdit_Phone">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_Email">
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 30px;
background: transparent; 
border: none;
</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="lineEdit_Email">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 45px;
border-radius: 50px; 
border: 2px solid #1E0342;
background-color: #A0DEFF;
color: #1E0342;
</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_Home">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/images/home-button.png);
font-weight: bold;
font-size: 50px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
font-size: 50px;
image: url(:/images/images/back_unactive.png);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Next">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/images/next-button.png);
font-weight: bold;
font-size: 50px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="styleSheet">
      <string notr="true">color: #1E0342;
font-weight: bold;
font-size: 20px;
border: none;
</string>
     </property>
     <property name="text">
      <string>Show password</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
